cmd/infracost/testdata/diff_with_config_file_compare_to/prior.json

```diff
@@ -6,7 +6,7 @@
       "name": "infracost/infracost/cmd/infracost/testdata/diff_with_config_file_compare_to/dev",
       "metadata": {
         "path": "dev",
-        "type": "terraform_dir",
+        "type": "terraform_cli",
```

Comment:
I think all the compare-to JSON files with this change can just be changed back to `terraform_dir`. It seems like this might have been a find and replace issue